src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/ReleasePermissionsController.cs

```diff
@@ -24,26 +24,26 @@ public class ReleasePermissionsController : ControllerBase
             _releasePermissionService = releasePermissionService;
         }
 
-        [HttpGet("releases/{releaseId}/contributors")]
-        public async Task<ActionResult<List<ContributorViewModel>>> ListReleaseContributors(
+        [HttpGet("releases/{releaseId}/roles")]
```

Comment:
You could constrain all of the Guid parameters with `:guid:` e.g. `{releaseId:guid}` in all the route changes made to this class.